exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -107,8 +106,10 @@ def invalid_url?(url)
           true
         end
 
-        def send_bytes(bytes) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
+        def send_bytes(bytes, timeout:) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
           retry_count = 0
+          timeout ||= @timeout
```

Comment:
It is possible for `timeout:` to be `nil` here (e.g. if `export` is called from `SimpleSpanProcessor`), in which case we use the configured timeout, which defaults to 10 seconds.